src/main/java/com/skypro/cart/model/Assortment.java

```diff
@@ -0,0 +1,23 @@
+package com.skypro.cart.model;
+
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+
+import java.util.List;
+
+
+@Repository
+public class Assortment {
+    private final static List<Item> items = new ArrayList<>(List.of(
+            new Item("Колбаса", 320),
+            new Item("Хлеб", 52),
+            new Item("Сыр", 220),
+            new Item("Яйца", 135),
+            new Item("Молоко", 74))
+    );
+
+    public static Item getItem(int index) {
```

Comment:
```suggestion
    public Item getItem(int index) {
```
и здесь тоже, зачем static-то, если бин?